core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProjectOrchestrator.scala

```diff
@@ -57,15 +61,22 @@ class BillingProjectOrchestrator(ctx: RawlsRequestContext,
    * Creates a "v2" billing project, using either Azure managed app coordinates or a Google Billing Account
    */
   def createBillingProjectV2(createProjectRequest: CreateRawlsV2BillingProjectFullRequest): Future[Unit] = {
+    def tagAndCaptureSentryEvent(e: Throwable): Unit = {
```

Comment:
This would be worth extracting into a separate class and renaming it `captureWorkspacesSentryEvent` or something along those lines if we want to alert more often on other failing operations we care about (workspace CRUD, spend reporting, etc.)

Reply:
I really like this style of alerting
Not needing to dig for the stack trace is great.
Alert => click => see problem

I'd be in favor or using this more generally - although we should probably try it out for realsies before going too ham.

Reply:
Agreed, let's get confidence in this approach before using it elsewhere.